pkg/kubelet/lifecycle/handlers.go

```diff
@@ -22,7 +22,7 @@ import (
 	"net/http"
 	"strconv"
 
-	"k8s.io/api/core/v1"
+	v1 "k8s.io/api/core/v1"
```

Comment:
nit: Not really necessary as `v1` is used throughout this file without the import alias

Reply:
This is automatically added by the `golang.org/x/tools/cmd/goimports` configured by my editor.

Reply:
Updated.